distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -1,3 +1,9 @@
+:title: Catalog REST Endpoint
+:type: endpoint
+:status: published
+:filename: catalog-rest-endpoint-contents.adoc
+:link: _catalog_rest_endpoint
+:summary: allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP.
 
 ==== Catalog REST Endpoint
```

Comment:
Need to remove this line as the template creates a section heading from the title property declared on line 1.